ecs/s3.conf

```diff
@@ -0,0 +1,23 @@
+[SERVICE]
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ecs_cluster ${ECS_CLUSTER}
+    Record ecs_task_arn ${ECS_TASK_ARN}
+    Record ecs_task_definition ${ECS_TASK_DEFINITION}
+
+[OUTPUT]
+    Name s3
+    Match *
+    bucket ${BUCKET}
+    region ${S3_REGION}
+    total_file_size 1M
+    upload_timeout 1m
+    use_put_object On
+    s3_key_format  /${ECS_CLUSTER}/${ECS_TASK_ID}/%Y/%m/%d/%H/%M/%S
```

Comment:
Yeah that's the public doc for FireLens V1, with our new feature it will change to what I noted in my comment, and these configs are for the new feature...